src/main/java/se/sundsvall/incident/integration/db/entity/IncidentEntity.java

```diff
@@ -72,15 +73,15 @@ public class IncidentEntity {
 	private Status status;
 
 	@ManyToOne
-	@JoinColumn(name = "category_id")
+	@JoinColumn(name = "category_id", foreignKey = @ForeignKey(name = "fk_incident_category"))
 	private CategoryEntity category;
 
 	@Builder.Default
 	@OneToMany(cascade = {CascadeType.PERSIST, CascadeType.MERGE})
 	@JoinTable(
```

Comment:
Replace this with. 

```
@JoinColumn(name = "incident_id", referencedColumnName = "id",
	foreignKey = @ForeignKey(name = "fk_incident_attachment_incident_id"))
```
a attachment cant live without a IncidentEntity, there is no reason to involve a joined table in this situation.